conans/client/generators/cmake_find_package.py

```diff
@@ -46,6 +46,10 @@ class CMakeFindPackageGenerator(GeneratorComponentsMixin, Generator):
                 {find_dependencies_block}
             endif()
         endif()
+
+        foreach(_BUILD_MODULE_PATH ${{{name}_BUILD_MODULES_PATHS}})
```

Comment:
maybe we have to add another type of build modules, one to be loaded before targets are created (as it right now) and this one adding build modules after targets are created

Reply:
Directly related to https://github.com/conan-io/conan/issues/7261